build/cmake/package_tester/modules/tests.sh

```diff
@@ -59,6 +59,14 @@ then
        then
           fail "value was not cleared correctly"
        fi
+       PYTHON_TARGZ_NAME="$(ls /build/packages | grep 'foundationdb-[0-9.]*\.tar\.gz' | sed 's/\.tar\.gz$//')"
```

Comment:
Couldn't this be an issue with the documentation. If the docs are built, they'll also be in a tar.gz file

Reply:
This regex won't match the docs tar.gz file, since the docs tar.gz file contains the substring "docs". That said if you have a more robust way to get the python package please let me know.